Assignment-1-July-18/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JSON Identifier</title>
+    <h1>Welcome to JSON Identifier</h1>
+    <link rel="stylesheet" href="style.css" />
+    <script>
+      function jsonChecker(text) {
+        try {
+          JSON.parse(text);
+          return true;
+        } catch {
+          return false;
+        }
+      }
+      function displayPostResponse(response)
+    {
+      response
+            .then((response) => {
+              return response.json();
+            })
+            .then((jsonResponse) => {
+              console.log(jsonResponse);
+              let uniqueId = jsonResponse.data.id;
+              document.getElementById(
+                "responseBox"
+              ).innerHTML = `Successfully added in DB with ID ${uniqueId}`;
+              console.log("Unique ID:", uniqueId);
+            })
+            .catch(() => {
+              document.getElementById("responseBox").innerHTML =
+                "Some Error Encountered";
+            });
+        } 
+    
+      function sendData() {        //Fetch API - POST usage
+        let input = document.getElementById("inputResponse").value;
+
+        if (jsonChecker(input)) {
+          let data = JSON.parse(input);
+
+          let response = fetch(
+            "https://dummy.restapiexample.com/api/v1/create",
+            {
+              method: "POST",
+              headers: {
+                "Content-Type": "application/json",
+              },
+              body: JSON.stringify(data),
+            }
+          );
+          displayPostResponse(response);//sending the promise to the Display Data function
+
+        }
+        else {
+          document.getElementById("responseBox").innerHTML =
+            "This can't be converted into JSON.";
+        }
+      }
+      function displayGetResponse(response)
+      {
+        response
+            .then((data) => {
+              console.log("After first then: ", data);
```

Comment:
remove all unnecessary console statements. show result and errors on the browser.